salt/elife-xpub/init.sls

```diff
@@ -56,29 +56,45 @@ elife-xpub-environment-variables-for-database-credentials:
             # local container
             export PGHOST=postgres
             export PGPORT=5432
-            export PGDATABASE=
+            export PGDATABASE=xpub
             export PGUSER=xpub
             export PGPASSWORD=
             {% endif %}
 
 elife-xpub-database-startup:
     cmd.run:
-        - name: {{ docker_compose }} up -d postgres
+        - name: |
+            {% if salt['elife.cfg']('cfn.outputs.RDSHost') %}
+            # remote RDS server
+            echo "RDS instance should already be started"
```

Comment:
Not sure if this is true...

Reply:
In any case, it does seem less confusing to only start the local postgres instance if there isn't a remote DB

Reply:
Although, when we do a `docker-compose up` it will be brought up again anyway. Will just sat there, empty until we find a way not to run it.